client/app/src/main/java/com/example/criengine/Database/GoogleBooksWrapper.java

```diff
@@ -84,7 +84,7 @@ protected Book doInBackground(String... isbns) {
                     }
                 }
 
-                Book book = new Book(null, title, authorsString, description, null, "Available");
+                Book book = new Book(null, null, title, authorsString, description, null, "Available");
```

Comment:
Change "Available" to "available" (lower case).
Activities rely on them being lower case.

For future advice, all status's should be lower case when stored in the object.